docs/src/guide/cli.md

```diff
@@ -0,0 +1,98 @@
+---
+outline: deep
+---
+
+# Command Line Interface
+
+The `fledge` command is an alias for `fledge create`.
+
+### Usage
+
+```bash
+fledge
+```
+
+### Arguments
+
+| Arguments       |                                                                                      |
+| --------------- | ------------------------------------------------------------------------------------ |
+| `[projectPath]` | Path where the project should be created. If omitted, the current directory is used. |
+
+### Options
+
+| Options                       |                                 |
+| ----------------------------- | ------------------------------- |
+| `-s, --source <stacksSource>` | Use the specified stacks source |
+| `-a, --auth <auth>`           | Use authorization key           |
+| `-c, --clear-cache`           | Clear stacks source cache       |
+
+### Example
+
+```bash
+fledge create my-new-project # or fledge my-new-project -c -s ./my-stack-of-choice
+```
+
+::: tip
+For detailed guidance on how to configure your own stack, please refer to the [configuration](/config) section.
+:::
+
+## Fledge Set
```

Comment:
This should be Config.

The CLI provides a sub program (config) which has two commands: `set` and `get`